backend/clubs/views.py

```diff
@@ -4679,7 +4679,7 @@ def retrieve(self, *args, **kwargs):
         return Response(data)
 
     @action(detail=True, methods=["post"])
-    def send_emails(self, *args, **kwargs):
+    def send_emails(self, request, *args, **kwargs):
```

Comment:
Does the request object need to be passed in?

Reply:
You're right, I don't have to. I can use self.request instead without passing in request